### ISSUE TITLE
Linker Error on Ada

### ISSUE THREAD

**Issue by JMetalian** [patriciogonzalezvivo/glslViewer#260]
Hi, I followed the instructions for installing the glslViewer. However, I'm constantly getting a linker error on Windows with Visual Studio 19:
![image](https://user-images.githubusercontent.com/43638551/152502885-27afe134-50df-472a-b90b-dd71b6ecd2c0.png)
![image](https://user-images.githubusercontent.com/43638551/152503182-f9232cc3-86c6-4699-b708-fbac4f0ebdac.png)
I have all of the requirements which shouldn't be an issue. Are there any dedicated WSDK version or anything ? Thanks

**Comment by Sonamaker1** [comment]
I had the same error and one possible solution is to do a few changes to shaders.h (to get it to compile on Windows with Visual Studio 2019) and then click rebuild solution.

Not saying my changes are a good idea, but it gave me something I could compile without the build failing.

**Changes made to shader.h:**

`static const GLchar* vertexShaderSrc = R"END(`

`static const GLchar* sdfFragShaderSrc = R"END(`

`static const GLchar* defaultFragShaderSrc = R"END(`

basically, the reverse of this commit on another repo (but only changing shaders.h):

https://github.com/tangrams/fontstash-es/commit/90786db9aa52b5072fc0df2f0ae317a354166948?diff=split#diff-1e19baf8b5675157cdd1dd33789420bd8510a05f216dd6c799b3a022f9e1d416

**Comment by patriciogonzalezvivo** [comment]
hi! Sorry for this troubles, just pushed a fix following you recommendations @Sonamaker1 . Do you mind confirming that works on windows?
Both glslViewer and `deps/ada` submodule have changed, please pull both, recompile and tell me if that works.

Thank you

**Comment by Sonamaker1** [comment]
> Both glslViewer and `deps/ada` submodule have changed, please pull both, recompile and tell me if that works.
It worked! Thanks for the update 😄

**Comment by JMetalian** [comment]
> hi! Sorry for this troubles, just pushed a fix following you recommendations @Sonamaker1 . Do you mind confirming that works on windows? Both glslViewer and `deps/ada` submodule have changed, please pull both, recompile and tell me if that works.
> 
> Thank you

Thank you so much. It is fixed. I wonder why I didn't try to fix it.
